src/Models/EventModel.php

```
@@ -238,49 +241,63 @@ public static function sendEvent(EventModel $event)
-            // Insert each time slot into the `event-tijd` table
-            $sqlEventTime = "INSERT INTO `event-tijd` (Event_ID, Land, Plaats, Straatnaam, Huisnummer, Postcode, Datum, BeginTijd, EindTijd, Sector) 
-                            VALUES (:eventID, :Land, :Plaats, :Straatnaam, :Huisnummer, :Postcode, :date, :BeginTijd, :EindTijd, :Sector)";
+            // Insert each time slot into the `event_tijd` table
+            $sqlEventTime = "INSERT INTO `event_tijd` 
+                             (event_id, land, plaatsnaam, straatnaam, huisnummer, postcode, datum, begin_tijd, eind_tijd) 
+                             VALUES (:eventID, :Land, :Plaats, :Straatnaam, :Huisnummer, :Postcode, :date, :BeginTijd, :EindTijd)";
 
             $stmtEventTime = $db->prepare($sqlEventTime);
 
-            foreach ($event->eventTime as $timeSlot) {
+            foreach ($event->eventTime as $i => $slot) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$i'. (undefined)

(UnusedLocalVariable)